libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -143,6 +144,16 @@ void AP_DDS_Client::main_loop(void)
         return;
     }
     GCS_SEND_TEXT(MAV_SEVERITY_INFO,"DDS Client: Initialization passed");
+
+    rcl_interfaces_msg_Log log;
+    update_topic(log.stamp);
+    strcpy(log.name, NODE_NAME);
```

Comment:
These need to get folded into the macro so the information is where the log call showed up